src/lib/__tests__/sortMenu-test.js

```diff
@@ -1,15 +1,19 @@
 import sortMenu from '../sortMenu';
 
 describe('sortMenu', () => {
-  it('modifies original array to follow the group, then name order provided', () => {
+  it('returns a new array of sorted children', () => {
```

Comment:
yes! much more functional :)

Modifying things in-place is almost always a bad idea